import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/ImportAdapter.java

```diff
@@ -157,15 +163,12 @@ private static IDicomWebClient configureDefaultDicomWebClient(
       GoogleCredentials credentials,
       Flags flags) {
     IDicomWebClient defaultCstoreDicomWebClient;
+    log.debug("--stow_overwrite set to " + (flags.useStowOverwrite ? "true" : "false"));
```

Comment:
nit: instead of having these debug statements in various places, we could have one function that logs the flags state for all of the flags in one place / function.

Reply:
Added issue #143 for this. Removed the debug statement for now.